communication guidelines.md

```diff
@@ -0,0 +1,77 @@
+## Purpose:
+[GitHub Actions](https://docs.github.com/en/actions/learn-github-actions) gives you the flexibility to build an automated software development lifecycle workflow. With [GitHub Actions for Azure](https://docs.microsoft.com/en-gb/azure/developer/github/) you can create workflows that you can set up in your repository to build, test, package, release and deploy to Azure
+This document is to set communication guidelines, processes for making GitHub Actions ready for the external community. We want to set a robust framework for communication, and governance for anything related to Azure Actions, for enabling both internal and external users to be always updated on any change in this area.
+
+## Background information needed:
+1. Microsoft open source program office’s protocols and definitions.
+2. We would use GitHub aliases for all the communication
+3. All the communication (except for a few cases) we would want all communication to be public Do we ever need private communication? Is there anything proprietary? 
```

Comment:
Should we remove these questions? And put the closure in this document itself?

Reply:
I think the 4th point covers these questions, so we can combine 3 and 4?

Reply:
For 1st point, added the link to guidelines
For your question on 2nd point, I tried to add GH alias in Teams, but could not. The Vice Versa seems possible (adding MS alias to GH Channel though the channel Admin's approval). And I believe anyone who would be a contributor, or part of the community would have a GH alias. Then we should continue using GH Alias for communication. Thoughts?
Combined 3 & 4, and removed the questions.